### ISSUE TITLE
Django 2.0 replaced django.core.urlresolvers with django.urls

### ISSUE THREAD

**Issue by jayvdb** [mgrp/django-distill#7]
`setup.py` & `requirements.txt`do not pin the version of Django, and Django 2.0 is out, and Django distill is broken.

```py
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/django_distill/management/commands/distill-local.py", line 14, in <module>
    from django_distill.renderer import (run_collectstatic, render_to_dir)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/django_distill/renderer.py", line 17, in <module>
    from django.core.urlresolvers import reverse
ModuleNotFoundError: No module named 'django.core.urlresolvers'
```

Refer to "django.core.urlresolvers" entry on their [deprecation](https://docs.djangoproject.com/en/2.0/internals/deprecation/) doc.

**Comment by meeb** [comment]
Thanks for the report. This is fixed in the newly published 0.9 release.

**Comment by junihoj** [comment]
Change the django.core.urlresolvers import reverse to django.urls import reverse.
